fe/fe-core/src/main/java/org/apache/doris/alter/RollupJobV2.java

```diff
@@ -540,7 +540,11 @@ protected boolean cancelImpl(String errMsg) {
 
     private void cancelInternal() {
         // clear tasks if has
-        AgentTaskQueue.removeBatchTask(rollupBatchTask, TTaskType.ALTER);
+        if (rollupBatchTask != null) {
```

Comment:
If this is a common case, better not print warn log

Reply:
rollupbatch task should not be null？it should be empty？how to produce it？it seems a little serious

Reply:
> rollup batch task should not be null？it should be empty？how to reproduce it？it seems a little serious rollup batch task now will be always initialized, so it is so strange

Seem this job object is read from `JournalEntity`.

Reply:
> rollup batch task should not be null？it should be empty？how to reproduce it？it seems a little serious rollup batch task now will be always initialized, so it is so strange

maybe in deserialization， rollup batch task will be null ?

Reply:
Maybe it is because we write sc/rollup job as AlterJobV2, this base class not have `rollupBatchTask`/`schemaChangeBatchTask`

Reply:
> > rollup batch task should not be null？it should be empty？how to reproduce it？it seems a little serious rollup batch task now will be always initialized, so it is so strange
> 
> maybe in deserialization， rollup batch task will be null ?

the rollup batch task not the serialization/deserialization filed。 so I think it maybe other proble

Reply:
> Maybe it is because we write sc/rollup job as AlterJobV2, this base class not have `rollupBatchTask`/`schemaChangeBatchTask`

hi @BiteTheDDDDt  is there a way to reproduce this bug。 AlterJobV2 is abstract class，once created，it must be RollupJobV2 or SchemaChangeJobV2， so I don't think we find the real cause.